x/concentrated-liquidity/testcontracts/contract-sources/counter/.cargo/config

```diff
@@ -0,0 +1,8 @@
+[alias]
```

Comment:
Note: most of the remaining diff past this point is source code for the CW contract used for testing. I only included it here as a reference and to ensure tests can be extended in the future by other contributors (black-box wasm is hard to work with)